README.md

```diff
@@ -1,2 +1,8 @@
 # neuroscience-ai-reading-course
 Notes for the Neuroscience &amp; AI Reading Course (SEM-I 2020-21) at BITS Pilani Goa Campus
+
+## Syntactic Structures in Language models
+1. [Assessing BERT's syntactic abilities - Goldberg](./jasleen_2017A7PS0077G/BERT-Goldberg%20paper%20overview.md)
+2. [What does BERT learn about the structure of the language - Sagot](./jasleen_2017A7PS0077G/BERT-Sagot%20paper%20overview.md)
+
```

Comment:
The Transfer_LM paper is missing? Is this intentional?

Reply:
The work done in the paper was not satisfactory enough for some parts. I have still added it though now.